src/inc/corprof.idl

```diff
@@ -629,11 +629,15 @@ typedef enum
 
     COR_PRF_HIGH_BASIC_GC                           = 0x00000010,
 
+    // Enables the MovedReferences/MovedReferences2 callback for compacting GCs only.
```

Comment:
In the future if we had a GC mode that wasn't named 'compacting' but it still moved objects I assume you would want MovedReferences there too right?

Reply:
Yes, but so are you asking me to update the comment to future proof it? Sorry, I didn't understand.

Reply:
I don't care if you update the comment, as long as we know what is expected : ) Thanks!